src/Pages/jongjin/Main/Main.js

```diff
@@ -1,13 +1,258 @@
 import React, { Component } from 'react';
+import Comment from './Comment/Comment'
+import jongjin from '../../../images/jongjin/종진.jpg'
+import Ironman from '../../../images/jongjin/아이언맨.jpg'
+import Thor from '../../../images/jongjin/토르.jpg'
+import Wanda from '../../../images/jongjin/스칼렛위치.jpg'
+import Hulk from '../../../images/jongjin/헐크.png'
+import Widow from '../../../images/jongjin/블랙위도우.jpg'
+import Thanos from '../../../images/jongjin/타노스.png'
+import Spider from '../../../images/jongjin/스파이더.jpg'
+import Antman from '../../../images/jongjin/앤트맨.jpg'
+import Captain from '../../../images/jongjin/캡틴아메리카.jpg'
+import Doctor from '../../../images/jongjin/닥터스트레인지.png'
+import BlackPanther from '../../../images/jongjin/블랙팬서.jpg'
+import Marble from '../../../images/jongjin/마블.png'
+import SpiderManpid from '../../../images/jongjin/스파이더맨.jpg'
+import './Main.scss'
+
+class MainJJ extends Component {
+
+    state = {
+        text: "",
+        user: "Ironman ",
+        commentList: [],
+        isLike: true,
+        like: 99
+    }
+
+    handleInputChange = (e) => {
+        this.setState({
+            text: e.target.value
+        });
+    }
+
+    handleKeyPress = (e) => {
+        if(e.key === 'Enter') {
+            this.handleClick();
+        }
+    }
+
+    handleClick = () => {
+        this.setState({
+            commentList: this.state.commentList.concat(this.state.text),
+            text: ''
+        })
+    }
+    
+    handleLike = () => {
+        this.setState({
+            isLike: !this.state.isLike
+        })
+        this.handleLikenumUp();
+    }
```

Comment:
```suggestion
    handleLike = () => {
        this.setState({
            isLike: !this.state.isLike
        }, () => this.handleLikenumUp();)
    }
```
- setState 함수는 비동기로 처리되어 지기 때문에 바뀐 state 값으로 함수를 실행시키고 싶은 경우 위와 같이 콜백함수(함수의 인자로 들어가는 함수)를 사용해서 처리해주셔야 합니다.
- setState, 비동기 / 동기 , callback 키워드로 학습해보시기 바랍니다 :)

Reply:
비동기 개념 학습해서 적용시켜보도록 하겠습니다.